### DISCUSSION TITLE
Using Xamarin/MAUI controls

### DISCUSSION THREAD

**Discussion by lylerolleman** [AvaloniaUI/Avalonia#12834] [Q&A]
I was wondering if there was a nice way of using Xamarin or MAUI controls in Avalonia

I know about `NativeControlHost` and if push comes to shove I'm sure I could convert the renderers/handlers into such, but I'd much rather have a way of working with the abstract control. 

Ex. if I wanted to use MAUI's `ActivityIndicator`, or other third party controls

**Comment by timunie** [comment]
I guess better to port these controls to Avalonia if possible. Check awsome avalonia for libs that have these already (maybe)
https://github.com/AvaloniaCommunity/awesome-avalonia

**Reply by lylerolleman** [reply]
While I agree that would be ideal, I don't always have the time to do it, or I might want to have native style controls, and would like a shortcut for it. 

Awesome Avalonia has some good stuff but is still a pretty small catalog of controls compared to more mature frameworks
